README.md

```diff
@@ -146,6 +146,38 @@ custom:
 ```
 This will remove all folders within the installed requirements that match
 the names in `slimPatterns`
+
+### Lamba Layer
+Another method for dealing with large dependencies is to put them into a
+[Lambda Layer](https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html).
+Simply add the `layer` option to the configuration.
+```yaml
+custom:
+  pythonRequirements:
+    layer: true
+```
+The requirements will be zipped up and a layer will be created automatically.
+Now just add the reference to the functions that will use the layer.
+```yaml
+functions:
+  hello:
+    handler: handler.hello
+    layers:
+      - {Ref: PythonRequirementsLambdaLayer}
+```
+If the layer requires additional or custom configuration, add them onto the `layer` option.
+```yaml
+custom:
+  pythonRequirements:
+    layer:
+      name: ${self:provider.stage}-layerName
+      description: Python requirements lamba layer
+      compatibleRuntimes:
+        - python3.7
```

Comment:
This should be set to the service runtime by default if the user uses `layer: true`

Reply:
Done!